frontend/src/component/admin/groups/EditGroup/EditGroup.tsx

```diff
@@ -143,15 +156,20 @@ export const EditGroup = ({
                 handleCancel={handleCancel}
                 mode={EDIT}
             >
-                <Button
-                    type='submit'
-                    variant='contained'
-                    color='primary'
-                    disabled={!isValid}
-                    data-testid={UG_SAVE_BTN_ID}
-                >
-                    Save
-                </Button>
+                <Tooltip title={scimEnabled ? scimGroupTooltip : ''} arrow>
+                    <div>
+                        <Button
+                            type='submit'
+                            variant='contained'
+                            color='primary'
+                            disabled={scimEnabled || !isValid}
+                            data-testid={UG_SAVE_BTN_ID}
+                            title='test'
```

Comment:
Intentional?

Reply:
Absolutely not intentional. Great catch!

Reply:
Addressed here: https://github.com/Unleash/unleash/pull/6866/commits/30a7594a4e88c5abdc9f1cc704fbc1a0bd0d0f18#diff-c8cdde5c0a91012fb86a001167f6ed6d15025c6dd501e109d4b4d36d06fef69eL167